ezgui/src/text.rs

```diff
@@ -298,6 +298,9 @@ impl Text {
             max_width = max_width.max(line_dims.width);
         }
 
+        let spaceholder = Polygon::rectangle(max_width, y);
```

Comment:
I don't know if this is reasonable, but it seems to work.

We have an invisible placeholder polygon that fills the dimensions, so that any sizing logic (like here: https://github.com/dabreegster/abstreet/blob/master/ezgui/src/widgets/button.rs#L395 ) maintains a constant line height regardless of the actual shape of the text within.